03-loops.md

```diff
@@ -412,8 +412,7 @@ for idx = 1:3
 end
 ~~~
 
-Remember that to run our script, we simply type in its name in the
-command line:
+Save the `analyze.m` script as a new script called `analyze_loops.m`. To do this, we go to the `Editor` tab, click on the arrow below `Save`, and then on `Save As...`. Remember that to run our script, we simply type in its name in the command line:
```

Comment:
Purely pedantic request: split this long line into several shorter lines, preferably using [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/).